slices/fontconfig-config.yaml

```diff
@@ -0,0 +1,32 @@
+package: fontconfig-config
+
+slices:
+  config:
+    essential:
+      - fonts-dejavu-core_fonts
+    contents:
+      /etc/fonts/conf.d/10-hinting-slight.conf:
+      /etc/fonts/conf.d/10-scale-bitmap-fonts.conf:
+      /etc/fonts/conf.d/10-sub-pixel-rgb.conf:
+      /etc/fonts/conf.d/10-yes-antialias.conf:
+      /etc/fonts/conf.d/11-lcdfilter-default.conf:
+      /etc/fonts/conf.d/20-unhint-small-vera.conf:
+      /etc/fonts/conf.d/30-metric-aliases.conf:
+      /etc/fonts/conf.d/40-nonlatin.conf:
+      /etc/fonts/conf.d/45-generic.conf:
+      /etc/fonts/conf.d/45-latin.conf:
+      /etc/fonts/conf.d/48-spacing.conf:
+      /etc/fonts/conf.d/49-sansserif.conf:
+      /etc/fonts/conf.d/50-user.conf:
+      /etc/fonts/conf.d/51-local.conf:
+      /etc/fonts/conf.d/60-generic.conf:
+      /etc/fonts/conf.d/60-latin.conf:
+      /etc/fonts/conf.d/65-fonts-persian.conf:
+      /etc/fonts/conf.d/65-nonlatin.conf:
+      /etc/fonts/conf.d/69-unifont.conf:
+      /etc/fonts/conf.d/70-no-bitmaps-except-emoji.conf:
+      /etc/fonts/conf.d/80-delicious.conf:
+      /etc/fonts/conf.d/90-synthetic.conf:
+      /etc/fonts/fonts.conf:
+      /usr/share/fontconfig/conf.avail/*.conf:
+      /usr/share/xml/fontconfig/fonts.dtd:
```

Comment:
can you add here a slice for the copyright file? (this a required thing now for all new slices since chisel is now supporting `essential` field at top level):

```yaml
  copyright:
    contents:
      /usr/share/doc/fontconfig-config/copyright:
```